PullRequestCreated/PullRequestCreated.ts

```diff
@@ -16,6 +22,25 @@ class PullRequestCreated extends OctokitAction {
   private shouldCreateIssue(pr: PullRequest): boolean {
     return pr != null && !JIRA_ISSUE_PATTERN.test(pr.title);
   }
+
+  private async newIssueTypeAndParent(pr: PullRequest): Promise<Issue> {
+    const mentionnedIssues = this.findMentionnedIssues(pr);
+    console.log(`Found ${mentionnedIssues.length} mentionned issues: ${mentionnedIssues}`);
+    let parent = mentionnedIssues.length === 1 ? await this.jira.getIssue(mentionnedIssues[0]) : null;
```

Comment:
This logic does not respect "first Epic if there are multiple issues mentioned"

Reply:
This is a separate PR: #106